.github/workflows/translation-test.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       run: |
         sudo apt-get update
         sudo apt-get install ninja-build
-        pip install dace
+        pip install git+git://github.com/spcl/dace@master
```

Comment:
consider using a specific revision ID here.

Reply:
I'd argue that there is no need. This is just to pass the tests, since the fixes didn't get released yet. Shouldn't rely on the repo, but the released version instead.